certified-connectors/Replicon/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2403 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Replicon",
+    "description": "Replicon is a self-driving professional services automation solution that uses advanced technology to analyze real-time data, and then delivers recommendations to help keep projects, resources, and finances on track. The connector allows you to continuously sync projects, tasks, and resource assignments from Project Online to Replicon and time data from Replicon to Project Online.",
+    "version": "1.0",
+    "contact": {
+      "name": "Replicon Support",
+      "url": "https://www.replicon.com/customerzone/contact-support/",
+      "email": "support@replicon.com"
+    }
+  },
+  "host": "global.replicon.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/services/ProjectService1.svc/BulkGetProjectDetails3": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ProjectDetailsResponse"
+            }
+          },
+          "default": {
+            "description": "default"
```

Comment:
@repl-anand-vikram Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger.